bloom-housing/bloom
# fix: unit accordion radio button not showing default value

# Pull Request Template

## Issue Overview

This PR addresses #2372 


## Description

When testing this I did see that changing the `Do you want to show unit types or individual units?` radio was already working to enable/disable the units accordion. What is fixed here is that the radio button was not loading the set value either on first open or after a refresh in the listing form.

Also fixed a UI issue in the accordion with the square feet value.

## How Can This Be Tested/Reviewed?

Ensure a listing loads the correct value on first open and after a refresh, and defaults to individual units.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
